### ISSUE TITLE
semicolons: 'always' triggers on multiline selectors

### ISSUE THREAD

**Issue by forshtreter** [rossPatton/stylint#162]
```stylus
.page {
    &__header,
    &__footer {
        ...
    }
}
```
In this case Stylint 1.0.9 warns that semicolon is missing after `&__header,`

**Comment by rossPatton** [comment]
should be good now in 1.0.10

**Comment by SimenB** [comment]
@rossPatton This is still buggy. Maybe not this specific case, but I'm getting a lot of false positives. It seems to trigger if the outer one has too many characters...
Create new issue perhaps?

This fails with `missing semicolon`
```stylus
.abcdefghij {
    .active {
        color: red;
    }
}
```

while this is good
```stylus
.abcdefghi {
    .active {
        color: red;
    }
}
```

The difference is that the first one goes all the way to `j`, while the second stops at `i`

**Comment by rossPatton** [comment]
should be addressed in 1.1.0
